tests/utils/index.js

```diff
@@ -63,6 +63,9 @@ const MINIMUM_BROWSER_VERSION = '90';
 const KUBECTL_CONTEXT = `-n ${PROJECT_NAME} --context k3d-${PROJECT_NAME}`;
 const cookieJar = rpn.jar();
 
+// Cookies from the jar will be included on Node `fetch` calls
+global.fetch = require('fetch-cookie').default(global.fetch, cookieJar);
```

Comment:
After some tinkering and thinking, I realized we are in a pretty stable place for authentication in cht-datasource. 

Webapp (online) and the admin app are using the _remote adapter_ which makes `fetch` calls from the browser. These `fetch` calls can be made relative to the "current page" (so the URL does not need to be provided) and they inherit the cookies for the browser session (so we don't need to worry about authentication).

Sentinel and api are using the _local adapter_ which is just `Pouch` under the hood and the injected Pouch instances are already authenticated.

---

However, our integration tests (and any future 3rd-party usage of cht-datasource) need to provide a root URL to use for calling the api server (done via  the `getRemoteDataContext` function).  This all works great.  Where things get tricky for the tests is the NodeJS implementation of the `fetch` api does not supply cookies from the `Set-Cookie` of previous responses when it makes new requests (this happens automatically in the browser environment).  So, the choices were either update the cht-datasource code to make it possible to inject the session cookie just so the tests would work, or get a bit creative with `fetch` during the test runs to be sure the cookies are there when `fetch` is called.  I have opted for the latter option for now. It keeps the impl code the most clean.  This will probably need to be re-visited in the future, either when we go to replace all our `request-promise-native` with `fetch` or when we release cht-datasource for 3rd-party consumption and we need a more proper way to do authentication.  But, for now, this seems like a reasonable approach especially since we can just piggy-back on our existing rpn cookie jar....